src/app/pages/journal/journal.page.html

```diff
@@ -2,7 +2,7 @@
     <ion-toolbar mode="ios">
         <ion-title id="page-title">Journal - DES 4</ion-title>
         <ion-buttons id="back-button" slot="start">
-            <ion-back-button></ion-back-button>
+            <ion-back-button id="ion-back-button"></ion-back-button>
```

Comment:
Would suggest changing to "journal-back-button" as this type of naming might conflict with other back buttons in future.

Reply:
I have changed the id to a more appropriate name